.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
What do you think of asking if they're willing to contribute a fix?

Reply:
Sounds great. I'll add it under the possible solution section.